ballerina/client.bal

```diff
@@ -156,3 +163,8 @@ isolated function nativeBatchExecute(Client sqlClient, string[]|sql:Parameterize
 returns sql:ExecutionResult[]|sql:Error = @java:Method {
     'class: "io.ballerina.stdlib.java.jdbc.nativeimpl.ExecuteProcessor"
 } external;
+
+isolated function isRequestGeneratedKeysSupportInBatchExecute(Options? options) returns boolean {
+    return (options is () || (options is Options &&
+                (options.requestGeneratedKeys == ALL || options.requestGeneratedKeys == BATCH_EXECUTE)));
```

Comment:
Wouldnt it be easier to test the negative scenario? `options.requestGeneratedKeys` can only be execute

Reply:
Updated in fceb82c